buildSrc/opensearch-hadoop-version.properties

```diff
@@ -1,4 +1,4 @@
-opensearch_hadoop        = 1.0.1
+opensearch_hadoop        = 1.1.0
 opensearch      = 2.12.0
 lucene          = 9.5.0-snapshot-a4ef70f
```

Comment:
we might want to update to the newest version of Lucene given that opensearch is now pointing to `2.12`?

Reply:
Have changed it to `9.9.2` to match `2.12.0`: https: //github.com/opensearch-project/OpenSearch/blob/2.12.0/buildSrc/version.properties